habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -408,7 +409,23 @@ def _load_navmesh(self, ep_info):
             base_dir = osp.join(*ep_info.scene_id.split("/")[:2])
 
         navmesh_path = osp.join(base_dir, "navmeshes", scene_name + ".navmesh")
-        self.pathfinder.load_nav_mesh(navmesh_path)
+        load_success = self.pathfinder.load_nav_mesh(navmesh_path)
```

Comment:
Can you just check if the path exists instead? There might some overhead with calling this function?

Reply:
thanks, that makes sense!